src/hammer-vlsi/hammer_vlsi/cli_driver.py

```diff
@@ -386,13 +386,25 @@ def action(driver: HammerDriver, append_error_func: Callable[[str], None]) -> Op
                 else:
                     post_load_func_checked(driver)
                 success, output = driver.run_drc(extra_hooks)
+                if not success:
```

Comment:
Long-term we should probably DRY this stuff

Reply:
Also SRAM generator is missing these checks but it's probably not that important.

Reply:
Filed #457